### PR TITLE
storing gifs in a folder

### PR DESCRIPTION
**Pull request by YongYi-coding**
@BintLopez @misternu @scottyschup Hello!!! I have the gifs store inside a directory named thank_you_gifs which is located inside the images directory. I think this solution is better than the other one where fun gif links are stored in an array. Let me know what you guys think!

I also center the log in page both horizontally and vertically. Before it was only vertically center. Which one is better? @jel535s @BintLopez  I have not made any styles to mobile screen yet. I think Nicole you said that we should probably remove the fun gif on mobile screen?

Screen Shot 2021-04-03 at 11 03 04 PM

Also I don't know why I have to bundle install and bundle update sq_mini_racer every time I make a new branch from the trunk.

### CODE REVIEW

**Review by misternu — APPROVED** [review]

File: app/views/layouts/devise.html.erb [hunk 1]
```diff
@@ -23,6 +23,9 @@
         <%= yield %>
         <a id="sqreen-badge" title="Realtime application protection" href="https://www.sqreen.com/?utm_source=badge"><img style="width:109px;height:36px" src="https://s3-eu-west-1.amazonaws.com/sqreen-assets/badges/20171107/sqreen-light-badge.svg" alt="Sqreen | Runtime Application Protection" /></a>
       </div>
+      <div class="devise-form container mx-0">
+        <%= image_tag get_random_gif, width: "370" %>
```
Comment (BintLopez) [inline app/views/layouts/devise.html.erb:27]: let's set the width in CSS

**Comment by YongYi-coding** [comment]
@BintLopez I put some CSS styling so that the gif stays under the log in
Screen Shot 2021-04-04 at 10 15 31 PM

**Comment by YongYi-coding** [comment]
@BintLopez When the screen width is 500px or under, no gif
Screen Shot 2021-04-04 at 10 16 15 PM

**Review by BintLopez — APPROVED** [review]
🦑 this looks great!

**Pull request merged**